pkg/sql/sqlbase/structured.go

```diff
@@ -584,6 +584,25 @@ func (desc *TableDescriptor) AllNonDropIndexes() []*IndexDescriptor {
 	return indexes
 }
 
+// AllPublicAndNonPublicChecks returns all the checks, including those being
+// added in the mutations.
+func (desc *TableDescriptor) AllPublicAndNonPublicChecks() []*TableDescriptor_CheckConstraint {
```

Comment:
Need a better name here, since it sounds the same as the existing `AllChecks` method. I wouldn't know which to use.

Reply:
I updated it so that constraints are either "active" (should be enforced for writes) or "inactive" (queued but invisible to writes). I'm open to better suggestions.

Reply:
Seems reasonable.